code/datums/status_effects/debuffs/rust_corruption.dm

```diff
@@ -0,0 +1,16 @@
+/datum/status_effect/rust_corruption
+	alert_type = null
+	id = "rust_turf_effects"
+	tick_interval = 2 SECONDS
+	remove_on_fullheal = TRUE
+	status_type = STATUS_EFFECT_REFRESH
+
+/datum/status_effect/rust_corruption/tick(seconds_between_ticks)
+	if(issilicon(owner))
+		var/mob/living/silicon/silicon_victim = owner
+		silicon_victim.adjustBruteLoss(10)
```

Comment:
1. You don't need to cast this. `adjustBruteLoss` is in the living level. 
2. Might as well make this scale by `seconds_between_ticks`.
```suggestion
		owner.adjustBruteLoss(5 * seconds_between_ticks)
```